### DISCUSSION TITLE
Setting up access manager

### DISCUSSION THREAD

**Discussion by chadht** [lithnet/access-manager#64] [General] [answered]
When setting up the Access Manager, for the AD step is there a specific PowerShell module that needs to be installed? I'm attempting to run the script but I get an error that the cmdlet is not recognized. 

Thanks,
Chad

**Comment by ryannewington** [comment]
The Active directory module is required for most of the scripts. Which one are you having issues with?

**Comment by chadht** [comment]
I’m attempting to run the first command: Add-DomainGroupMembershipPermissions (screenshot attached) I do not see the exact command on the MS ActiveDirectory PowerShell page<https://docs.microsoft.com/en-us/powershell/module/addsadministration/?view=win10-ps>. Should I try using Add-ADGroupMember or AddADPrincipalGroupMembership instead?

Thanks,
Chad Tidgewell
Technology Support | HCPSS
techsupport@hcpss.org<mailto:techsupport@hcpss.org> | Technology Resources<https://hcpss.instructure.com/courses/32959>


The information contained in this email transmission is privileged and/or confidential and is intended solely for the exclusive use of the individual addressee. If you are not the intended addressee, you are hereby notified that any retention, disclosure, or other use is strictly prohibited. If you have received this email in error, please contact the sender immediately and delete the material.

**Comment by ryannewington** [comment]
Hi Chad,

The screen shot didn't come through. Can you try attach it again?

**Comment by chadht** [comment]
Hi Ryan,

Here you go. Thanks.
[cid:image001.png@01D6D2FB.803639F0]

Thanks,
Chad Tidgewell
Technology Support | HCPSS
techsupport@hcpss.org<mailto:techsupport@hcpss.org> | Technology Resources<https://hcpss.instructure.com/courses/32959>


The information contained in this email transmission is privileged and/or confidential and is intended solely for the exclusive use of the individual addressee. If you are not the intended addressee, you are hereby notified that any retention, disclosure, or other use is strictly prohibited. If you have received this email in error, please contact the sender immediately and delete the material.

**Comment by ryannewington** [comment]
It's getting dropped by GitHub when you use the email reply function I think. Email it to support@lithnet.io

**Comment by chadht** [comment, marked as answer]
Hi Ryan,

Here you go. I may have figured it out, I now see where I need to add the AD module for PowerShell to features for the server I'm using. Thanks again for taking a look.

[cid:image001.png@01D6D2FD.041433C0]

Chad Tidgewell
Technology Support | HCPSS
techsupport@hcpss.org<mailto:techsupport@hcpss.org> | Technology Resources<https://hcpss.instructure.com/courses/32959>


The information contained in this email transmission is privileged and/or confidential and is intended solely for the exclusive use of the individual addressee. If you are not the intended addressee, you are hereby notified that any retention, disclosure, or other use is strictly prohibited. If you have received this email in error, please contact the sender immediately and delete the material.
